test/sort/processor-spec.ts

```diff
@@ -0,0 +1,144 @@
+import 'jest-extended';
+import { DataEntity, AnyObject } from '@terascope/job-components';
+import { WorkerTestHarness } from 'teraslice-test-harness';
+import DataWindow from '../../asset/src/__lib/data-window';
+
+const testData = [
+    {
+        id: 1
+    },
+    {
+        id: 3
+    },
+    {
+        id: 2
+    }
+];
+
+const dataWindows: DataWindow[] = [];
+for (let i = 0; i < 3; i++) {
+    // @ts-expect-error
+    const dataArray = Array(10).fill().map(() => {
```

Comment:
*Minor*, you can use `times` from `@terascope/utils`.

```suggestion
    const dataArray = times(10, () => {
```